qcore/nhm.py

```diff
@@ -32,50 +60,205 @@ class NHMFault:
         fault surface trace (lon, lat)
     """
 
-    def __init__(self, entry: List[str]):
-        """Creates an NHMFault instance from the given NHM text.
+    def __init__(
```

Comment:
This can be replaced with member fields and the @dataclass decorator.
The main difference would be removing 2 of the 3 copies of each variable.
If we add other stuff to the initialiser then we'll need to put it back though.